test/functional/helpers.lua

```diff
@@ -438,4 +439,5 @@ return {
   curbufmeths = curbufmeths,
   curwinmeths = curwinmeths,
   curtabmeths = curtabmeths,
+  ffi = ffi,
```

Comment:
This doesn't need to be served by the helpers module; test suites should just `require('ffi')` themselves.